lib/storage/index_db.go

```diff
@@ -65,6 +65,9 @@ type indexDB struct {
 	// The counter for time series which were re-populated from previous indexDB after the rotation.
 	timeseriesRepopulated uint64
 
+	// The counter for daily index records which were re-populated from current indexDB to the new indexDB before the rotation.
+	dailyIndexRecordsRepopulated uint64
```

Comment:
it is unclear when this counter increases:
1. When current indexDB is used to re-populate entires for new indexDB object
2. Or when current indexDB gets new entries from previous IndexDB object.